### PR TITLE
cleanup camera malfunction alert

### PR DESCRIPTION
**Pull request by deanlee**

### CODE REVIEW

File: selfdrive/ui/ui.cc [hunk 1]
```diff
@@ -283,17 +283,16 @@ void ui_update(UIState *s) {
...
+    const uint64_t frame_delayed = (s->sm)->frame - frame_pkt;
+    if ((frame_pkt > s->started_frame && frame_delayed > 5*UI_FREQ) ||
+        frame_delayed > 35*UI_FREQ) {
       // controls is fine, but rear camera is lagging or died
       s->scene.alert_text1 = "Camera Malfunction";
       s->scene.alert_text2 = "Contact Support";
       s->scene.alert_size = cereal::ControlsState::AlertSize::FULL;
       s->status = STATUS_DISENGAGED;
       s->sound->stop();
-      ui_draw_vision_alert(s, s->scene.alert_size, s->status,
```
Comment (deanlee) [inline selfdrive/ui/ui.cc:295]: we don't need to call ui_draw_vision_alert here

Comment (adeebshihadeh) [reply selfdrive/ui/ui.cc:295]: it's gated on vision_connected. https://github.com/commaai/openpilot/blob/master/selfdrive/ui/paint.cc#L541

Comment (deanlee) [reply selfdrive/ui/ui.cc:295]: oic

**Comment by deanlee** [comment]
Move ui_draw_vision_alert() out of  ui_draw_vision(),put it before nvgEndFrame(), ensure that the alert will always be displayed.

File: selfdrive/ui/ui.cc (hunk 1 above)
Comment (deanlee) [reply selfdrive/ui/ui.cc:295]: We should not call ui_draw_vision_alert outside of beginframe/endframe.

**Comment by ZwX1616** [comment]
looks fine to me. works also.
waiting for approval from @adeebshihadeh

**Pull request merged**